### PR TITLE
[zh-cn] Resync debug-service page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/tasks/debug/debug-application/debug-service.md [hunk 1]
```diff
@@ -217,11 +219,13 @@ What would happen if you tried to access a non-existent Service?  If
...
-## Service 是否存在？
+## Service 是否存在？   {#does-the-service-exist}
 
-细心的读者会注意到我们实际上尚未创建 Service -这是有意而为之。 这一步有时会被遗忘，这是首先要检查的步骤。
+细心的读者会注意到我们实际上尚未创建 Service -这是有意而为之。
+这一步有时会被遗忘，这是首先要检查的步骤。
 
-那么，如果我尝试访问不存在的 Service 会怎样？ 假设你有另一个 Pod 通过名称匹配到 Service ，你将得到类似结果：
+那么，如果我尝试访问不存在的 Service 会怎样？ 
+假设你有另一个 Pod 通过名称匹配到 Service ，你将得到类似结果：
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/debug/debug-application/debug-service.md:228]:
```suggestion
假设你有另一个 Pod 通过名称匹配到 Service，你将得到类似结果：
```

File: content/zh-cn/docs/tasks/debug/debug-application/debug-service.md [hunk 2]
```diff
@@ -22,10 +23,11 @@ get no response when you try to access it.  This document will hopefully help
 you to figure out what's going wrong.
 -->
 对于新安装的 Kubernetes，经常出现的问题是 Service 无法正常运行。 你已经通过
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/debug/debug-application/debug-service.md:25]:
```suggestion
对于新安装的 Kubernetes，经常出现的问题是 Service 无法正常运行。你已经通过
```

File: content/zh-cn/docs/tasks/debug/debug-application/debug-service.md [hunk 3]
```diff
@@ -314,7 +318,6 @@ traffic to `hostnames-*` Pods, these need to be reviewed.
 
 Please refer to [Network Policies](/docs/concepts/services-networking/network-policies/) for more details.
 -->
-
 ## 是否存在影响目标 Pod 的网络策略入站规则？
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/debug/debug-application/debug-service.md:321]:
```suggestion
## 是否存在影响目标 Pod 的网络策略入站规则？   {#any-network-policy-ingress-rules-affecting-the-target-pods}
```

File: content/zh-cn/docs/tasks/debug/debug-application/debug-service.md [hunk 4]
```diff
@@ -624,38 +630,43 @@ Earlier you saw that the Pods were running.  You can re-check that:
...
+
 <!--
 The `-l app=hostnames` argument is a label selector configured on the Service.
 
 The "AGE" column says that these Pods are about an hour old, which implies that
 they are running fine and not crashing.
+-->
+`-l app=hostnames` 参数是在 Service 上配置的标签选择器。
 
+"AGE" 列表明这些 Pod 已经启动一个小时了，这意味着它们运行良好，而未崩溃。
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/debug/debug-application/debug-service.md:649]:
```suggestion
“AGE” 列表明这些 Pod 已经启动一个小时了，这意味着它们运行良好，而未崩溃。
```

**Review by Sea-n — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/tasks/debug/debug-application/debug-service.md (hunk 3 above)
Comment (Sea-n) [reply content/zh-cn/docs/tasks/debug/debug-application/debug-service.md:321]: Okay, now I know it's unnecessary to add not-in-use link anchors.

File: content/zh-cn/docs/tasks/debug/debug-application/debug-service.md [hunk 5]
```diff
@@ -217,11 +219,13 @@ What would happen if you tried to access a non-existent Service?  If
 you have another Pod that consumes this Service by name you would get
 something like:
 -->
-## Service 是否存在？
+## Service 是否存在？   {#does-the-service-exist}
 
-细心的读者会注意到我们实际上尚未创建 Service -这是有意而为之。 这一步有时会被遗忘，这是首先要检查的步骤。
+细心的读者会注意到我们实际上尚未创建 Service -这是有意而为之。
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/debug/debug-application/debug-service.md:224]:
```suggestion
细心的读者会注意到我们实际上尚未创建 Service —— 这是有意而为之。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/tasks/debug/debug-application/debug-service.md (hunk 3 above)
Comment (howieyuen) [reply content/zh-cn/docs/tasks/debug/debug-application/debug-service.md:321]: how do I know if these anchors in-use or not?

Comment (tengqm) [reply content/zh-cn/docs/tasks/debug/debug-application/debug-service.md:321]: You can make a good guess when you manually input a lengthy, verbose anchor ID. If you are referencing a target using things like `#any-network-policy-ingress-rules-affecting-the-target-pods`, you know something is wrong.

Comment (howieyuen) [reply content/zh-cn/docs/tasks/debug/debug-application/debug-service.md:321]: interesting, got it 👍

**Pull request merged**